manifests/init.pp

```diff
@@ -6,33 +6,26 @@
 #
 # Mark Stanislav <mstanislav@duosecurity.com>
 class duo_unix (
-  $usage = '',
-  $ikey = '',
-  $skey = '',
-  $host = '',
-  $group = '',
-  $http_proxy = '',
-  $send_gecos = '',
-  $fallback_local_ip = 'no',
-  $failmode = 'safe',
-  $pushinfo = 'no',
-  $autopush = 'no',
-  $motd = 'no',
-  $prompts = '3',
-  $accept_env_factor = 'no',
-  $manage_ssh = true,
-  $manage_pam = true,
-  $manage_repo = true,
-  $pam_unix_control = 'requisite',
-  $package_version = 'installed',
+  String[20] $ikey,
+  String[40] $skey,
+  Stdlib::Host $host,
+  Enum['login', 'pam'] $usage = 'pam',
+  Optional[String[1]] $group = undef,
+  Optional[String[1]] $http_proxy = undef,
+  Enum['yes', 'no'] $send_gecos = 'no',
+  Enum['yes', 'no'] $fallback_local_ip = 'no',
+  Enum['safe', 'secure'] $failmode = 'safe',
+  Enum['yes', 'no'] $pushinfo = 'yes',
```

Comment:
https://duo.com/docs/duounix says the default pushinfo is 'no', why change here?

Reply:
Thanks, fixed.